src/Features/Core/Portable/Shared/Extensions/ISymbolExtensions_2.cs

```diff
@@ -129,6 +129,9 @@ public static Glyph GetGlyph(this ISymbol symbol)
                 case SymbolKind.PointerType:
                     return ((IPointerTypeSymbol)symbol).PointedAtType.GetGlyph();
 
+                case SymbolKind.FunctionPointer:
+                    return Glyph.Intrinsic;
```

Comment:
I chose a glyph that seemed appropriate here. If there's a better suggestion, please let me know :)

Reply:
wfm